### ISSUE TITLE
Consider pulling the schemas repo on-prem

### ISSUE THREAD

**Issue by amrc-benmorrow** [AMRC-FactoryPlus/amrc-connectivity-stack#212]
There is quite a lot to be said for pulling the schemas repo on-prem, rather than contacting public Github directly. This would need these actions:

* Update the default repo pulls in the ACS values file to create a `schemas` repo.
* The repo mirrors from the values file are currently configured in `clusters.js` in service-setup. This needs moving into a new file and the result passing into both `clusters.js` and `manager.js` so the Manager can be informed of the repo UUID.
* Update the Manager to accept the schema repo UUID in the environment.
* If the Manager currently relies on any Github functionality, this needs reworking to use Git directly.

~We need to decide soon if this is practical for v3.0.0.~

**Comment by AlexGodbehere** [comment]
It probably makes more sense to store these in the Config Store now?

**Comment by amrc-benmorrow** [comment]
Yes, I agree schemas should end up in the ConfigDB. However we need to master that information somewhere, and I'm not sure stuffing them all into the ACS repo is a good idea. ACS works perfectly well with a completely different set of schemas, if someone wants to do that.

Maybe we leave them in their own repo, and rather than mirroring we have that repo build another image like the edge-helm image which pushes the preset schemas into the ConfigDB? We could also let people supply their own service-setup images to preload their own schemas or whatever else.

For private schemas a service pulling from AMRC github (or other corporate version control) and updating the ConfigDB would be useful.
